### DISCUSSION TITLE
Soft-wrapping at arbitrary line length

### DISCUSSION THREAD

**Discussion by glow12121** [orbitalquark/textadept#353] [Q&A] [answered]
Is it possible to add the settings which would:

a. always soft-wrap the buffer
b. do it at a certain line length instead of the edge of the typing area?

**Comment by orbitalquark** [comment, marked as answer]
Unfortunately, Scintilla (Textadept's editing component) does not 
support this, so neither is a possibility. Sorry about that :(

**Reply by glow12121** [reply]
Thanks for the answer... Left a feature request on Scintilla's tracker.
